src/expandable-section/styles.scss

```diff
@@ -2,16 +2,17 @@
  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  SPDX-License-Identifier: Apache-2.0
 */
-
+@use 'sass:map';
```

Comment:
What's this for?

Reply:
Right, this can be removed, I'm not using it in the end.